runtime/server_http_request.go

```diff
@@ -96,13 +83,8 @@ func (req *ServerHTTPRequest) start(endpoint string, handler string) {
 		/* coverage ignore next line */
 		return
 	}
-
-	req.EndpointName = endpoint
-	req.HandlerName = handler
 	req.started = true
 	req.startTime = time.Now()
-
-	req.metrics.requestRecvd.Inc(1)
```

Comment:
Let's keep the recvd metric here in the `start()` method.

Reply:
Yes, as we discussed offline.